salesagility/SuiteCRM
# Added studio.php file in Target Lists module

## Description

Took Accounts' studio.php file changed contents to match Target List.

Navigate to studio and try to find Target List module.


## Motivation and Context

Change is needed to alter target list layout and add fields ect.
## How To Test This

Navigate to studio, find Target List module make changes and check if reflected in module views.

## Types of changes


### Final checklist
